packages/react/src/client.ts

```diff
@@ -57,7 +56,7 @@ export function createClient<
       dehydrateOptions: {
         shouldDehydrateQuery: (query) =>
           query.cacheTime !== 0 &&
-          query.queryHash !== JSON.stringify(signerQueryKey()),
+          (query.queryKey[0] as { persist?: boolean }).persist !== false,
```

Comment:
nice abstraction! a quick comment here that `persist` can be added to query keys to control the behavior could be nice.

Reply:
Yep! sg!